buildSrc/src/main/kotlin/com/datadog/gradle/config/KotlinConfig.kt

```diff
@@ -26,24 +24,4 @@ fun Project.kotlinConfig(
             languageVersion.set(KotlinVersion.KOTLIN_1_7)
         }
     }
-
-    val moduleName = this@kotlinConfig.name
-    val javaAgentJar = File(File(rootDir, "libs"), "dd-java-agent-1.26.1.jar")
-    taskConfig<AndroidUnitTest> {
-        if (environment["DD_INTEGRATION_JUNIT_5_ENABLED"] == "true") {
-            val variant = variantName.substringBeforeLast("UnitTest")
-
-            // set the `env` tag for the test spans
-            environment("DD_ENV", "ci")
-            // add custom tags based on the module and variant (debug/release, flavors, …)
-            environment("DD_TAGS", "test.module:$moduleName,test.variant:$variant")
```

Comment:
It seems that we are losing these tags? I can see that `test.configuration.variant` was added in the `gitlab-ci.yml` file, but the name of the tag is different, and `test.module` tag is completely missing. Are these 2 added automatically by the agent by some chance?

Reply:
Yes, the module tag is added automatically, its value is slightly different though (the path of the Gradle task).

I changed the name of the variant tag to make it a part of the [test configuration](https://docs.datadoghq.com/tests/#custom-configurations) - that way the same test executed via a different task will be considered a different test case in the CI Vis backend. I wasn't exactly sure if this is a possible case - if not, I can revert the tag to its original name.

Reply:
ok, clear enough, let's try to have these changes then.